flow/designs/asap7/gcd/config.mk

```diff
@@ -1,37 +1,13 @@
 $(info [INFO-FLOW] GCD Design)
 
-DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
-DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
-
-export DESIGN_NAME            = gcd
-export DESIGN_NICKNAME        = gcd
-export DESIGN                 = gcd
-
 export PLATFORM               = asap7
 
-export VERILOG_FILES          = $(sort $(wildcard $(abspath $(DESIGN_DIR)/../../src/$(DESIGN))/*.v))
-export SDC_FILE               = $(DESIGN_DIR)/constraint.sdc
-
 export CORNER                ?= BC
+export DESIGN_NAME            = gcd
+export DESIGN_NICKNAME        = gcd
```

Comment:
the default is 
export DESIGN_NICKNAME ?= $(DESIGN_NAME)

so there is no need to specify nickname when it matches name